good-enough-practices-for-scientific-computing.tex

```diff
@@ -132,7 +129,7 @@ \section{Data Management}\label{sec:data}
   format that is inefficient to store or process, transform it for
   storage with a lossless, well-documented procedure.  Equally, don't
   duplicate contents of stable, long-lived repositories (i.e., don't
-  clone GenBank).
+  download the entire DNA sequence database GenBank).
```

Comment:
I get the point, but we actually download the entire database from GenBank to our server on a regular basis, to run local bulk searches.